.github/workflows/UpdateBaselineVersion.yaml

```diff
@@ -0,0 +1,29 @@
+name: Update Baseline Version
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "55 03 * * *" # Run at 03:55 UTC on Tuesday
```

Comment:
Either the comment of the value is wrong.

Reply:
Good catch :)